src/cargo/ops/cargo_install.rs

```diff
@@ -1,7 +1,10 @@
-use std::collections::{BTreeMap, BTreeSet};
-use std::path::{Path, PathBuf};
-use std::sync::Arc;
-use std::{env, fs};
+use std::{
+    collections::HashSet,
+    collections::{BTreeMap, BTreeSet},
+    env, fs,
+    path::{Path, PathBuf},
+    sync::Arc,
+};
```

Comment:
Could these refactorings of imports be left to a later PR?

Reply:
Yes. I only did it as jetbrains only "import with similar" does that refactoring and I was too lazy to do it by hand. I will fix.